tfhe/src/integer/server_key/radix/tests.rs

```diff
@@ -1129,3 +1131,65 @@ fn integer_smart_scalar_sub(param: Parameters) {
         }
     }
 }
+
+fn integer_unchecked_scalar_decomposition_overflow(param: Parameters) {
```

Comment:
no test for scalar_sub that was also fixed ?

Reply:
Yes it's tested starting at line 1159.

Reply:
right thanks !